roles/cobbler/tasks/apt_systems.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Install cobbler
+  apt:
+    name: "{{ cobbler_package }}"
+    state: latest
+  register: install_cobbler
+
+- name: Install syslinux
```

Comment:
Should we put these extra packages in a list and use with_items to install them?  These next few tasks seem very repetitive.